### DISCUSSION TITLE
This is a very noob friendly and welcoming community.

### DISCUSSION THREAD

**Discussion by G-Dikik** [uBlockOrigin/uAssets#14463] [General]
Oh. No, not at all actually what is wrong with you people.

**Comment by okiehsch** [comment]
Your problem is what?
I added and closed the issue with a "fix" as you can see in the issue.
You then update your lists and if it does not work for you go ahead and tell us.

**Reply by okiehsch** [reply]
I could reproduce your issue with a default setup and the filter I added fixes the issue on my end.
You indicate in your report that you used a default setup.
Which I highly doubt looking at all the filter lists you enabled.

**Reply by G-Dikik** [reply]
I would tell that its not fixed if did not just toxicly block me from contributing or instant close a not fixed issue ticked.

**Reply by G-Dikik** [reply]
i also updated all filters by emptying the cache first, which did nothing

**Reply by okiehsch** [reply]
Have you tried with a default setup?
Closing an issue does not mean that it is "gone".

**Reply by G-Dikik** [reply]
About your indication of me lying. I did a "is it my fault check" with a default firefox in a sandbox. But gave you my normal setup debug info.

**Reply by okiehsch** [reply]
Have you tried with a default setup after I added the filter?

**Comment by G-Dikik** [comment]
**Finally** for now. The fix did not work for me in the first place because of my personal filters. Now it seems to be fine as you claimend.

**In summary if you @okiehsch would follow your own rules of** 

> Be as clear as possible: nobody can read mind, and nobody is looking at your issue over your shoulder.

this whole conversation would have been much easier for everyone.

**Reply by okiehsch** [reply]
Fair enough about the be as clear as possible part.

**Reply by G-Dikik** [reply]
thanks

**Comment by ellamacrow98** [comment]
Hello all[.](https://advancedcardetailing.com/) I am new here
